assets/wizards/popups/views/analytics/Chart.js

```diff
@@ -16,15 +16,16 @@ import {
 const Chart = ( { data } ) => (
 	<div className="newspack-campaigns-wizard-analytics__chart">
 		<ResponsiveContainer width={ '100%' } height={ 300 }>
-			<AreaChart data={ data } margin={ { top: 5, right: 20, bottom: 5, left: -15 } }>
+			<AreaChart data={ data } margin={ { top: 8, right: 16, bottom: 8, left: -32 } }>
```

Comment:
With large enough values, such a small offset is cropping the y axis' values:

![image](https://user-images.githubusercontent.com/7383192/121337491-e1d48280-c91c-11eb-81ad-1b8851d6e92a.png)

Reply:
I clearly don't have large enough values on my dev site 😅
Restored margins in da9aadd